src/centralized_telescope.rs

```diff
@@ -240,7 +240,7 @@ impl Proof {
     /// Round{v, t, x_1, ..., x_u)} such that:
     /// - ∀i ∈ [0, u-1], H0(x_i+1) ∈ bins[H1(...H1(H1(v, t), x_1), ..., x_i)]
     /// - H2(H1(... H1((H1(v, t), x_1), ..., x_u)) = true
-    /// otherwise None
+    ///     otherwise None
```

Comment:
Clippy warning: 
```shell
warning: doc list item without indentation
   --> src/centralized_telescope.rs:243:9
    |
243 |     /// otherwise None
    |         ^
```